ci/deploy/multi-country-killswitch-pipeline.yaml

```diff
@@ -0,0 +1,182 @@
+---
+apiVersion: concourse.govsvc.uk/v1beta1
+kind: Pipeline
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: killswitch
+spec:
+  exposed: false
+  paused: true
+  config:
+
+    resource_types:
+      - name: paas-s3
+        type: docker-image
+        source:
+          repository: governmentpaas/s3-resource
+          tag: latest
+      - name: slack-notification
+        type: docker-image
+        source:
+          repository: cfcommunity/slack-notification-resource
+
+    resources:
+      - name: verify-slack
+        type: slack-notification
+        source:
+          url: ((slack-webhook))
```

Comment:
On the GSP, I think this needs to be `((slack-notification.url))`, which is a sealed secret in this directory.
Also, I think that webhook slack url may be for #eidas-alerts

Reply:
👍